app/assets/javascripts/public/templates/bookmarks/delete.jst.ejs

```diff
@@ -0,0 +1,31 @@
+<div class="ui basic modal">
+  <div class="ui icon header">
+    <i class="trash icon"></i>
+    Delete Bookmark?
+  </div>
+  <div class="content">
+    <p>Are you sure you want to delete this bookmark?</p>
```

Comment:
Could this deletion modal be a reusable thing?

Reply:
Probably could, but since I've only got one right now I don't think it's worth the hassle to generalize

Reply:
I think I counted two